filebeat/_meta/fields.common.yml

```diff
@@ -176,3 +176,74 @@
           description: >
             An array of Kafka header strings for this message, in the form
             "<key>: <value>".
+
+- key: threat
+  title: threat
```

Comment:
To keep the default_field list in the mapping in check this should set `default_field: false`. This helps Filebeat stay below the `indices.query.bool.max_clause_count` [^1] when users perform searches. In Beats 8.x we changed `default_field: false` to be the default so this is only needed on backports into 7.x.

[^1]: https://www.elastic.co/guide/en/elasticsearch/reference/7.17/search-settings.html#search-settings